selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -265,7 +267,7 @@ DevicePanel::DevicePanel(SettingsWindow *parent) : ListWidget(parent) {
   });
 
   // power buttons
-  QHBoxLayout *power_layout = new QHBoxLayout();
+  power_layout = new QHBoxLayout();
```

Comment:
I saw we also did this on our prior split, but I am unsure why do we need it (lack of the context) so i wonder if we really needed it ?

Reply:
Let's revert this for now. This was needed for Force Offroad, but it's not covered in this PR yet.